locksmith/__tests__/controllers/v2/metadataController.test.ts

```diff
@@ -78,6 +78,18 @@ describe('Metadata v2 endpoints for locksmith', () => {
     expect(userMetadataResponse.body).toStrictEqual({
       metadata,
     })
+
+    const getUserMetadataResponse = await request(app)
+      .get(`/v2/api/metadata/locks/${lockAddress}/users/${address}`)
```

Comment:
```suggestion
      .get(`/v2/api/metadata/100/locks/${lockAddress}/users/${address}`)
```

where `100` is the network.